packages/FPauth-core/FPauth-core.1.0.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+license: "MIT"
+synopsis: "Easy authentication system for Dream framework"
+maintainer: "Mikhail Geine <mike.geine@gmail.com>"
+authors: [
+  "Mikhail Geine <mike.geine@gmail.com>"
+  "Pavel Argentov <argentoff@gmail.com>"
+]
+homepage: "https://github.com/mikeGEINE/FPauth"
+bug-reports: "https://github.com/mikeGEINE/FPauth/issues"
+depends: [
+  "dream" {>= "1.0.0~alpha3"}
+  "ocaml" {>= "4.12.0"}
+  "dune" {>= "2.7"}
+  "lwt_ppx" {>= "2.0.3"}
+  "base"
+  "alcotest" {with-test}
+  "bisect_ppx" {with-test & >= "2.5.0"}
+  "odoc" {with-doc}
+]
```

Comment:
```suggestion
]
conflicts: [
  "base-nnp"
]
```

Reply:
I'm actually not sure for now how to fix this, so it's probably better to constrain against nnp now. Thank you!)